techs/zetapack/factions/persian/units/elephant/elephant.xml

```diff
@@ -7,7 +7,7 @@
 	<parameters>
 		<size value="2"/>
 		<height value="2"/>
-		<max-hp value="2700" regeneration="4"/>
+		<max-hp value="2700" regeneration="20"/>
```

Comment:
This seems like a very high regeneration rate, compared to all the units across ZetaPack.

I'm not sure how much we should be adjusting units yet in the original megapack before we balace out the new factions.

Aside from magic and tech, nothing has really been done yet with the other megapack factions. Romans of course I changed to get rid of the cows being produced at the forum, then food changed to grace, and increasing catapult range (across 3 different factions), and added splash damage to the turtle.

But now romans needs more changing because we weren't happy how temples for grace turned out, so I'm more open to modifying romans at this point to make sure it balances (which you've done below).

But I'm not going to vote this particular change up or down until @mathusummut or @KeithSammut provide some feedback too.

I would suggest adding splash damage to the elephant in this PR, as discussed in #127

Reply:
For me, the elephant is totally useless. I tried increasing its regen to see if that makes it more useful.

Adding splash might be a good idea, then I would definitely decrease the regen.

Reply:
The splash could be the same as the roman turtle? https://github.com/ZetaGlest/zetaglest-data/commit/d055702acb3d62298700928e1359b0f4bc0be0cc

Reply:
OK, I'll add that.

Reply:
and could also add it to the roman battering ram?